lighthouse-core/audits/audit.js

```diff
@@ -125,7 +125,12 @@ class Audit {
     if (displayValue === score) {
       displayValue = '';
     }
-
+    let auditDescription = audit.meta.description;
+    if (audit.meta.fallbackDescription) {
+      if (!result.rawValue || (typeof result.rawValue === 'number' && result.score < 100)) {
```

Comment:
Pushed a new commit, is that what you meant? Sorry, was a bit unsure when I review a second time!